source/Ansible-Protocol-Core/AmqpChannel.class.st

```diff
@@ -326,9 +326,7 @@ AmqpChannel >> initializeUsing: aChannelHandler [
 AmqpChannel >> mapEventHandlers [
 
 	handler mapEvent: protocolVersion channelCloseMethod to: self selector: #handleChannelClose:.
-	handler mapEvent: protocolVersion basicDeliverMethod to: self selector: #handleBasicDeliver:.
-	handler mapEvent: protocolVersion basicReturnMethod to: self selector: #handleBasicReturn:.
-	handler mapEvent: protocolVersion channelFlowMethod to: self selector: #handleChannelFlow:
+	handler mapEvent: protocolVersion basicDeliverMethod to: self selector: #handleBasicDeliver:
```

Comment:
I would love to understand this change, I don't get it. I'll for the next review

Reply:
I've removed the 2 last handlers because the methods are not implemented in the image, so if any of this appears it will produce a DNU.